lib/mixlib/install/backend/base.rb

```diff
@@ -189,6 +189,11 @@ def windows_artifact_fixup!(artifacts)
             end
           end
 
+          # We currently only support appx packages for 2016nano
+          artifacts.delete_if do |artifact|
+            artifact.url.end_with?(".appx") unless %w{2016nano}.include?(artifact.platform_version)
+          end
+
```

Comment:
I suppose we should if they are built. I added the inverse of this into another block.